inst/pages/11_taxonomic_information.qmd

```diff
@@ -329,29 +329,29 @@ references.
    regular clr (see above) but allows data with zeroes and avoids the
    need to add pseudocount [@Keshavan2010, @Martino2019].
 
-  * 'pa' presence/absence transformation ignores abundances and only
+  * 'pa' Presence/Absence transformation ignores abundances and only
    indicates whether the given feature is detected above the given
    threshold (default: 0). This simple transformation is relatively
    widely used in ecological research. It has shown good performance
    in microbiome-based classification performance [@Giliberti2022,
    Karwowska2024].
 
- * 'z' Z transformation scales data to zero mean and unit variance;
-   this is used to bring features (or samples) to more comparable
-   levels in terms of mean and scale of the values. This can enhance
+ * 'standardize' Standardize transformation scales data to zero mean and 
+   unit variance. This is used to bring features (or samples) to more 
+   comparable levels in terms of mean and scale of the values. This can enhance
```

Comment:
Add "(or Z-transformation)"

Reply:
I would prefer not referring to z-transformation any more since it has different common meaning.

Reply:
Ok, that is fine. I guess Z-score is relatively common though. The description already states clearly what the transformation does, but for someone Z-score might be more familiar term as "standardization" can also mean different things

Reply:
Can you point us to at least 1-3 references where other than us use z transformation in our meaning in microbiome analyses?

Reply:
With a quick search (I have not done that earlier; my point was that it might be good to list common synonyms from basic statistics even though they are not commonly used in microbiome research.)

- https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-016-0222-x/figures/2
- https://www.researchgate.net/publication/338713495_Diversity_of_Bacterial_Community_in_the_Oxygen_Minimum_Zones_of_Arabian_Sea_and_Bay_of_Bengal_as_Deduced_by_Illumina_Sequencing/figures?lo=1
. https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8250139/

Reply:
This also discusses "z-score" in the same meaning: https://en.wikipedia.org/wiki/Standard_score

The "z" has multiple meanings like "z score", "z value" "z factor" etc.
https://en.wikipedia.org/wiki/Z-score_(disambiguation)

Seems fine to refer to this also as z-score.